tests/test_ontology.py

```diff
@@ -342,3 +342,22 @@ def test_parsing_limits_bad_class(onto):
             """ 
         g.parse(data = annotated_ttl)
         onto.parse_oda_annotations(g)
+
+
+def test_get_requested_resources(onto):
+    g = rdf.Graph()
+    annotated_ttl = add_prefixes + """
+            oda:notebook1 oda:usesRequiredResource oda:MyS3 .
+            oda:MyS3 a oda:S3 .
+            oda:MyS3 oda:resourceBindingEnvVarName "My_S3_CREDENTIALS" .
+            oda:notebook2 oda:usesOptionalResource oda:MyComputeResource .
+            oda:MyComputeResource a oda:ComputeResource .
+            oda:MyComputeResource oda:resourceBindingEnvVarName "MY_COMPUTE_CREDENTIALS" .
```

Comment:
This contradicts what you define [in ontology PR](https://github.com/oda-hub/ontology/blob/8c042f5219eeb43e7109989b234957110097e78c/ontology.ttl#L170-L171) that the domain is only a StorageResource. I think you can broaden the domain there

Reply:
@dsavchenko , I have broaden the domain of resourceBindingEnvVarName property in ontology as you suggested